src/common/ownsql.cpp

```diff
@@ -278,7 +278,7 @@ int SqlQuery::prepare(const QByteArray &sql, bool allow_failure)
 static bool startsWithInsensitive(const QByteArray &a, const char *b)
 {
     size_t len = strlen(b);
-    return a.size() >= len && qstrnicmp(a.constData(), b, Utility::convertSizeToUint(len)) == 0;
+    return size_t(a.size()) >= len && qstrnicmp(a.constData(), b, Utility::convertSizeToUint(len)) == 0;
```

Comment:
Please avoid C-style casts. Better user a static_cast and then break that statement on two lines or such.